pages/en/lb4/includes/CLI-std-options.md

```diff
@@ -0,0 +1,11 @@
+{% if include.title != "no" %}
+**Standard options**
+{% endif %}
+`-h, --help`
```

Comment:
Do we need an extra line? 
When I was viewing it after clicking the "View" button, there's no new line before the `-h`.

Reply:
I copied the file from the lb3 folder, so if we need a new line before the `-h`, we'd need to change both files. What do you think?

Reply:
ok. If it works fine in loopback.io, then it's ok.  The "View" in github doesn't always show exactly on the web site.